setting_guideline/Stripe Setting Guideline.md

```diff
@@ -0,0 +1,128 @@
+# Stripe Payment Settings Instruction
+
+This guide will walk you through obtaining the necessary keys to integrate Stripe with your application.
+To get started, you'll need an active Stripe merchant account. If you don't have one yet, you can sign up at [Stripe](http://stripe.com/).
+
+---
+
+## Step 1: **Configure Stripe OAuth and Retrieve Client ID**
```

Comment:
Which Markdown viewer software you are using? The heading should be rendered as bold text already. No need to use `**` to make it bold.